modules/enet/enet_packet_peer.cpp

```diff
@@ -175,6 +175,12 @@ int ENetPacketPeer::get_channels() const {
 	return peer->channelCount;
 }
 
+int ENetPacketPeer::get_packet_flags() const {
+	ERR_FAIL_NULL_V(peer, ERR_UNCONFIGURED);
+	ERR_FAIL_COND_V(packet_queue.is_empty(), ERR_UNAVAILABLE);
```

Comment:
These are invalid returns, should be some flag, also don't think the `peer` part is necessary here